flake/dev/new-plugin.py

```diff
@@ -9,15 +9,18 @@
 default_nix_template = """{{ lib, ... }}:
 lib.nixvim.plugins.mkNeovimPlugin {{
   name = "{name}";
+  moduleName = "LUA_MODULE_NAME"; # TODO replace (or remove entirely if it is the same as `name`)
```

Comment:
```suggestion
  moduleName = "LUA_MODULE_NAME"; # TODO Remove if it is the same as `name`
```

The "replace" case should be handled by the generate script, right?

Reply:
I don't think IFD is relevant, since we aren't within an nix eval here. This is a python script used at runtime. If we needed to eval some nix we could use `nix-instanciate` or `nix eval` while running the script. If we needed to build something we could do that at runtime too.

But if your point was the script doesn't currently check `moduleName` is correct, then that's a perfectly good reason to keep the comment as-is. :+1: